### PR TITLE
New package: Samsung.TouchLogon version 2.0.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112018&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/TouchLogon/2.0.0/Samsung.TouchLogon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.TouchLogon
+PackageVersion: 2.0.0
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4590954&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
```
Comment (Trenly) [inline manifests/s/Samsung/TouchLogon/2.0.0/Samsung.TouchLogon.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
```

File: manifests/s/Samsung/TouchLogon/2.0.0/Samsung.TouchLogon.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4590954&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: Setup_x64-R.msi
+  InstallerSha256: 65513A4FBB2C84DF813A5DAAE3716F4BA18F56F5AFD8982EAA833BFFC394173D
+  ProductCode: '{AFF0220E-1F50-4B46-91BF-7812F71275F0}'
```
Comment (Trenly) [inline manifests/s/Samsung/TouchLogon/2.0.0/Samsung.TouchLogon.installer.yaml:15]:
```suggestion
  ProductCode: '{AFF0220E-1F50-4B46-91BF-7812F71275F0}'
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
  InstallerLocale: en-US
  Architecture: x86
  InstallerType: zip
  NestedInstallerType: msi
  NestedInstallerFiles:
  - RelativeFilePath: Setup_x86-R.msi
  InstallerSha256: 65513A4FBB2C84DF813A5DAAE3716F4BA18F56F5AFD8982EAA833BFFC394173D
  ProductCode: '{D1190F96-AF15-47B1-BCFD-F6EAE2D91EDE}'
```
Please also add the x86 installer

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**